docs/materials/day3/part1-ex2-matlab.md

```diff
@@ -0,0 +1,152 @@
+
+		:::console
+		username@build $ mcc -m -R -singleCompThread -R -nodisplay -R -nojvm matrix.m
+
+	The extra arguments to the `mcc` command are very important here. Matlab, by default, will run on as many CPUs as it can find. This can be a big problem  when running on someone else's computers, because your Matlab code might interfere with what the owner wants. The `-singleCompThread` option  compiles the code to run on a single CPU, avoiding this problem. In addition, the `-nodisplay` and `-nojvm` options turn off the display (which won't exist  where the code runs). 
+
+1.  To exit the interactive session, type `exit`
+
+1.  Now that you're back on the submit server, look at the files that were created by the Matlab compiler. Which one is the compiled binary?
+
+Matlab Runtime
+--------------
+
+The newly compiled binary will require the 2015b Matlab runtime to run. You can download the runtime from the Mathworks website and build it yourself, but to save time, for this exercise you can use the pre-built runtimes hosted by CHTC.
+
+1.  Download the 2015b Matlab runtime hosted by CHTC: 
+
+		:::console
+		username@learn $ wget http://proxy.chtc.wisc.edu/SQUID/2015b.tar.gz 
+
+Wrapper Script
+--------------
+
+Like the [OpenBUGS](/materials/day2/part3-ex4-prepackaged) example from earlier this morning, we will need a wrapper script to open the Matlab runtime and then run our compiled Matlab code. Our wrapper script will need to accomplish the following steps:
+
+-   Unpack the transferred runtime
+-   Set the environment variables
+-   Run our compiled matlab code
+
+Fortunately, the Matlab compiler has pre-written most of this wrapper script for us!
+
+1.  Take a look at `run_matrix.sh`. Which of the above steps do we need to add? Once you have an idea, move to the next step.
+
+1. We'll need to add commands to unpack the runtime (which will have been transferred with the job). Add this line to the beginning of the `run_matrix.sh` file, after `#!/bin/bash` and the comments, but before `exe_name=$0` : 
+
+		:::file
+		tar -xzf r2015b.tar.gz
+
+1. Look at `readme.txt` to determine what arguments our wrapper script requires. Once you have an idea, move to the next step.
+
+1. The name of the Matlab runtime directory is a required argument to the wrapper script `run_matrix.sh`. We'll have to do a little extra work to find out the name of that directory. Run this command
+
+		:::console
+		tar -tf r2015b.tar.gz
+
+1. The output of the previous command is a list of all the files in the tar.gz file. What is the name of the first folder of the path for each file? This is the name of the runtime directory, and the argument you should pass to `run_matrix.sh`.  
+
+Submitting the Job
+------------------
+
+1.  Copy an existing submit file into your current directory. The submit file we used for the [Open Bugs](part3-ex4-prepackaged.md) example would be a good candidate, as that example also used a wrapper script. 
```

Comment:
@ChristinaLK I was really confused about the broken link, too. It looks like you've moved the 'prepackaged' exercise to Tuesday and this matlab exercise is in Wednesday so we need to specify the path. If you accept this suggestion, I think it should fix everything.

```suggestion
1.  Copy an existing submit file into your current directory. The submit file we used for the [Open Bugs](/materials/day2/part3-ex4-prepackaged.md) example would be a good candidate, as that example also used a wrapper script. 
```